### ISSUE TITLE
conda qt package breaks compilation of cmake projects with find_package(Qt4)

### ISSUE THREAD

**Issue by thisch** [ContinuumIO/anaconda-issues#210]
If the conda qt package is installed, it is no longer possible to compile cmake projects which require Qt headers/libs. 

Consider this minimal CMakeLists.txt file

```cmake
cmake_minimum_required(VERSION 2.8.2)
find_package(Qt4)
message(STATUS "includes: ${QT4_INCLUDE_DIRS}")
```

``cmake .`` outputs

```
...
-- Looking for Q_WS_X11
-- Looking for Q_WS_X11 - found
-- Looking for Q_WS_WIN
-- Looking for Q_WS_WIN - not found
-- Looking for Q_WS_QWS
-- Looking for Q_WS_QWS - not found
-- Looking for Q_WS_MAC
-- Looking for Q_WS_MAC - not found
-- Could NOT find Qt4 (missing:  QT_MOC_EXECUTABLE QT_RCC_EXECUTABLE QT_UIC_EXECUTABLE) (found version "4.8.5")
-- includes:
....
```

**Comment by alexgmcm** [comment]
I'm also having this problem. It even occurs when I set the qmake path to /usr/bin/qmake (and check using which qmake)

**Comment by ccordoba12** [comment]
I'll try to fix it for the next Anaconda release.

**Comment by ccordoba12** [comment]
@alexgmcm, could you check again with Anaconda 2.2? We added a `qt.conf` to our Qt packages (in all platforms) that should fix this error.

**Comment by ccordoba12** [comment]
This is now fixed in Anaconda 2.2.

I already verified that that's the case, so I'm closing this issue.
